textord/tospace.cpp

```diff
@@ -932,6 +932,9 @@ ROW *Textord::make_prop_words(
   box_it.set_to_list (row->blob_list ());
   word_it.set_to_list (&words);
   bol = TRUE;
+  fuzzy_non = FALSE;
```

Comment:
These initializations look redundant to me. They are already initialized at declaration time near line 894.

Having said that, I find the mix of declaration time and non-declaration initializations needlessly complex and think all of these later initializations should get rolled back up into the declarations.